packages/EasyCodingStandard/bin/ecs

```diff
@@ -10,6 +11,10 @@ require_once __DIR__ . '/autoload.php';
 // performance boost
 gc_disable();
 
+$xdebug = new XdebugHandler('ecs');
+$xdebug->check();
+unset($xdebug);
```

Comment:
Maybe avoid the need to unset by simply not using a variable:

```
(new XdebugHandler('ecs'))->check();
```

Reply:
I don't know how exactly this works internally, so I used the README code: https://github.com/composer/xdebug-handler

Could you confirm the expected behavior is the same?

Reply:
Not sure when I'll have the time to test it. I'm about to leave from work and I don't have xdebug installed at home. But I'm quite confident it can't break with this change.

Reply:
Ok, fell free to test and send PR later if it bothers your.

I'll merge it as it is just to keep it consistent with common use.